nodeinfo/manager.go

```diff
@@ -65,16 +65,26 @@ type (
 	getBroadcastListFunc func() []string
 )
 
-var _nodeInfoHeightGauge = prometheus.NewGaugeVec(
-	prometheus.GaugeOpts{
-		Name: "iotex_node_info_height_gauge",
-		Help: "height info of node",
-	},
-	[]string{"address", "version"},
+var (
```

Comment:
one gauge is enough?

Reply:
it is for grafana transition. If we just add p2pID into existing gauge, would that cause data error on grafana? nodes with new version have 3 labels in the gauge, while nodes with current version still have 2 labels in the gauge

Reply:
Should work well using one gauge